### PR TITLE
fix bellow input emote refresh menu to toggle button

### LINKED ISSUE #396: 'Below Input' Emote Menu refreshes menu
I noticed this minor bug for the Emote Menu, in Appearance > Interface > Emote Menu Button Placement
When the menu is set to 'Below Input', the menu does not close when clicking the 7TV emote menu button after the menu is already open. Clicking the 7TV menu button refreshes the menu instead of closing it.
I did notice that when the settings menu is open, the emote menu does open and close normally when pressing the menu button.

When the emote menu is set to 'Regular', the menu button works as expected. This only happened when set to Below Input

### PR DESCRIPTION
**Pull request by Batyodie**
Fixes #396

The issue was related to toggling the emoji menu button when setting the interface "Bellow input".

Settings option:

image

How the menu was buggy before:

[screencast-www.twitch.tv-2023.04.12-21_04_53.webm](https://user-images.githubusercontent.com/61266300/231545965-fe0aaabe-213d-42e2-bd95-1f9a3c8753bf.webm)

How did it become:

[screencast-www.twitch.tv-2023.04.12-21_24_02.webm](https://user-images.githubusercontent.com/61266300/231551074-b7e9fddd-8691-4bfa-9bdb-e99d2ebb2cc8.webm)

### CODE REVIEW

File: src/site/twitch.tv/modules/emote-menu/EmoteMenu.vue [hunk 1]
```diff
@@ -186,6 +187,14 @@ watch(
 	{ immediate: true },
 );
 
+// in the "Bellow input" format, sometimes the positioning styles for the menu are not correct
+// when first opened.I manually force the position to be recalculated
+watch(ctx, (newVal) => {
+	if (newVal.open) {
+		uiFloatingMiddleware.value = [shift({ mainAxis: true, crossAxis: true })];
```
Comment (Batyodie) [inline src/site/twitch.tv/modules/emote-menu/EmoteMenu.vue:194]: There were some problems with the first launch of the menu window for setting the Bellow input. For some reason, the menu is not correctly positioned on the first click. I decided to add a watcher that monitors the opening and starts recalculating the position of the menu styles. Perhaps not the best solution. Can you tell me the best way to do this?

**Comment by ezlatsen** [comment]
> Fixes #396
> 
> The issue was related to toggling the emoji menu button when setting the interface "Bellow input".
> 
> Settings option:
> 
> image
> 
> How the menu was buggy before:
> 
> How did it become:

Yes, that was it exactly.
Thank you so much.

**Comment by Batyodie** [comment]
@AnatoleAM Hello. Can you take a look please. There was some kind of conflict, I tried to update it and possibly pulled extra commits.

**Comment by Batyodie** [comment]
@AnatoleAM Made rebase the current branch of the wizard, updated the readme

**Review by AnatoleAM — COMMENTED** [review]
This isn't actually a good fix. It's more of a band-aid; the problem stems from earlier up.

**Comment by AnatoleAM** [comment]
Closing as this is no longer an issue following indirect changes

**Pull request closed without merging**